### ISSUE TITLE
Add process to convert SCE to AnnData

### ISSUE THREAD

**Issue by allyhawkins** [AlexsLemonade/scpca-nf#226]
**Is your feature request related to a problem? Please describe.**

We want to be able to output the gene by cell counts matrices as AnnData objects stored as H5 files as well as output the SCE objects we already produce. Currently there are multiple SCE objects that get output as RDS files, the unfiltered, filtered, and processed RDS files. We probably want to have all of them also available as AnnData objects so that they are directly compatible with python. 

**Describe the solution you'd like**

We can create a process that converts RDS files with SCE objects to H5 files with AnnData objects and then run that process 3 times, once for each type of SCE file. Alternatively we could have a process that converts all three in one go using the same input that the QC report uses and then runs a script that converts all 3. I think I might favor the second option, but I'm curious if others have other ideas. 

Either way we will definitely need a script that takes RDS files as input and outputs H5 files using the `scpcaTools::sce_to_anndata` function. 

**Additional context**

We may need to break this up into additional issues as we start to work on this but I wanted to file this to get us started on adding AnnData conversion. We want to do this before we have our next big release and re-process projects.

**Comment by jashapiro** [comment]
I think for generality, it makes more sense to have the process convert one at a time. With the right set of transformations, it should not be too hard to take a channel with a form like `[[meta1, sce1a, sce1b, sce1c], [meta2, sce2a, ...]]` and convert it to `[[meta1, sce1a],[meta1, sce1b], [meta1, sce1c], [meta2, sce2a], ...]`

The code to do that transformation would look something like this:

```
my_ch.flatMap{[[it[0], it[1]], 
               [it[0], it[2]],
               [it[0], it[3]]
              ]}
```

We could then do single element transformations of a sce to anndata, and follow that with a `.groupTuple(size: 3)` step to put them back together. And then a step to flatten the resulting tuple... so in total, something like this:

```
sce_ch = in_ch.flatMap{[[it[0], it[1]], 
                        [it[0], it[2]],
                        [it[0], it[3]]
                       ]}

anndata_ch = sce_to_anndata(sce_ch)

out_ch = anndata_ch.groupTuple(size:3)
  .map{[it[0]] + it[1]}
```

**Comment by allyhawkins** [comment]
Closed by #350
